include/boost/graph/dijkstra_shortest_paths.hpp

```diff
@@ -589,31 +589,44 @@ namespace boost {
       std::vector<D> distance_map(n);
 
       detail::dijkstra_dispatch2
-        (g, s, choose_param(distance, make_iterator_property_map
-                            (distance_map.begin(), index_map,
-                             distance_map[0])),
+        (g, s_begin, s_end,
+         choose_param(distance, make_iterator_property_map
+                      (distance_map.begin(), index_map,
+                       distance_map[0])),
          weight, index_map, params);
     }
   } // namespace detail
 
-  // Named Parameter Variant
-  template <class VertexListGraph, class Param, class Tag, class Rest>
+  // Named parameter version for multiple start vertices
+  template <class VertexListGraph, class SourceInputIter, class Param,
+           class Tag, class Rest>
   inline void
   dijkstra_shortest_paths
     (const VertexListGraph& g,
-     typename graph_traits<VertexListGraph>::vertex_descriptor s,
+     SourceInputIter s_begin, SourceInputIter s_end,
      const bgl_named_params<Param,Tag,Rest>& params)
   {
     // Default for edge weight and vertex index map is to ask for them
     // from the graph.  Default for the visitor is null_visitor.
     detail::dijkstra_dispatch1
-      (g, s,
+      (g, s_begin, s_end,
        get_param(params, vertex_distance),
        choose_const_pmap(get_param(params, edge_weight), g, edge_weight),
        choose_const_pmap(get_param(params, vertex_index), g, vertex_index),
        params);
   }
 
+  // Named parameter version for a single start vertex
+  template <class VertexListGraph, class Param, class Tag, class Rest>
+  inline void
+  dijkstra_shortest_paths
+    (const VertexListGraph& g,
+     typename graph_traits<VertexListGraph>::vertex_descriptor s,
+     const bgl_named_params<Param,Tag,Rest>& params)
+  {
+    dijkstra_shortest_paths(g, &s, &s + 1, params);
+  }
+
```

Comment:
@Belcourt I think this is the overload that you might have overlooked that preserves the existing interface.